runtime/doc/options.txt

```diff
@@ -4386,6 +4386,17 @@ A jump table for the options with a short description can be found at |Q_op|.
 	default now.  This should work fine for most people, however if you
 	have any problem with it, try using on-the-spot style.
 
+			*'inchlsearch'* *'ihls'* *'noinchlsearch'* *'noihls'*
+'inchlsearch' 'ihls'	boolean	(default off)
+			global
+			{not in Vi}
+			{not available when compiled without the
+			|+extra_search| features}
+	While typing a search command, all matched strings are highlighted.
+	It works only when both 'incsearch' and 'hlsearch' are on. The type of
```

Comment:
The help text needs to be updated. The 'hlsearch' option is not needed
to enable this feature.